### PR TITLE
Retag vsphere cluster api provider images

### PR DESCRIPTION
**Pull request by QuentinBisson**

### CODE REVIEW

File: images.yaml [hunk 1]
```diff
@@ -209,6 +209,9 @@
     tag: v0.5.6
   - sha: 51cd47349de3232990b1d449f3d7d36f042ca0554a09f2d7feafb47dffb5c52d
     tag: v0.7.4
+- name: gcr.io/cluster-api-provider-vsphere/release/manager
+  patterns:
```
Comment (QuentinBisson) [inline images.yaml:213]:
```suggestion
  overrideRepoName: cluster-api-provider-vsphere-manager
  patterns:
```

**Comment by tfussell** [comment]
@QuentinBisson Should we drop the manager image since we're going to be building it from our fork? Retagging the rbac proxy still makes sense to me.

**Comment by QuentinBisson** [comment]
We can drop the manager for sure. Do you mind taking this PR over or close it and open a new one for the rbac proxy?

**Comment by tfussell** [comment]
> We can drop the manager for sure. Do you mind taking this PR over or close it and open a new one for the rbac proxy?

Sure, I can take it over.

**Review by anvddriesch — APPROVED** [review]

**Comment by tfussell** [comment]
Closing in favor of https://github.com/giantswarm/retagger/pull/621

**Pull request closed without merging**